.github/workflows/balena.yml

```diff
@@ -122,3 +127,32 @@ jobs:
               ((retry_count++))
             fi
           done
+          echo "::set-output name=sut-container-result::${result}"
+
+      - name: Stream logs from SUT container
+        id: sut-container-logs
+        if: ${{ github.event_name == 'pull_request' && github.event.action != 'closed'}}
+        run: |
+          set -uae
+
+          [[ '${{ env.VERBOSE }}' =~ on|On|Yes|yes|true|True ]] && set -x
+          balena login --token '${{ secrets.BALENACLOUD_API_KEY }}'
+          balena whoami
+
+          balena logs ${{ steps.find-test-device.outputs.balena_dut_uuid }} --service sut
```

Comment:
This will run indefinitely.

Reply:
It won't without the `--tail` flag. This will just be a one time command I intend to run after testing is finished to pull the logs